pkg/detectors/ldap/ldap.go

```diff
@@ -63,7 +65,11 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 				}
 
 				if verify {
-					s1.Verified = verifyLDAP(ctx, username[1], password[1], ldapURL)
+					err := verifyLDAP(username[1], password[1], ldapURL)
+					s1.Verified = err == nil
+					if !isErrDeterminate(err) {
+						s1.VerificationError = err
+					}
```

Comment:
**Nit:** The `s1.Verified = err == nil` looks weird, but I'm not sure this is better. 

```go
if err := verifyLDAP(username[1], password[1], ldapURL); err == nil {
    s1.Verified = true
} else if !isErrDeterminate(err) {
    s1.VerificationError = err
}
```

It's *technically* different behavior since we don't explicitly set `s1.Verified = false` for the `err != nil` case.